assemblers/org.osbuild.qemu

```diff
@@ -59,6 +59,10 @@ SCHEMA = """
     "type": "string",
     "enum": ["raw", "raw.xz", "qcow2", "vdi", "vmdk", "vpc", "vhdx"]
   },
+  "qcow2_compat": {
+    "description": "The compatibility version to use qcow2 images",
```

Comment:
I find this description hard to understand. Maybe sth more like `The qcow2-compatibility-version to use.`?

Reply:
There is a "for" missing. :/ I like your version, yeah. Alternatively "The qcow2 format version to use", but then the compatibility part is left out.

Reply:
I think @dvdhrm's suggestion makes the most sense while still being concise

Reply:
Your suggestion sounds good!